tests/testament/testamenthtml.nim

```diff
@@ -0,0 +1,300 @@
+import strutils
+
+proc htmlQuote*(raw: string): string =
+  if (raw.isNil):
+    return nil
+  result = raw
+  result = result.replace("&", "&amp;")
```

Comment:
Waiting for https://github.com/nim-lang/Nim/pull/6193 to be merged so that we can use the new `strutils.multiReplace`